ExampleMod/Content/Tiles/ExampleCrystal.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.ObjectData;
+
+namespace ExampleMod.Content.Tiles
+{
+    internal class ExampleCrystal : ModTile
+    {
+        public override void SetStaticDefaults()
+        {
+            TileObjectData.newTile.Width = 1;
+            TileObjectData.newTile.Height = 1;
+            TileObjectData.newTile.CoordinateWidth = 16;
+            TileObjectData.newTile.CoordinateHeights = new int[] { 16 };
+            TileObjectData.newTile.CoordinatePadding = 2;
+            TileObjectData.newTile.StyleHorizontal = true;
+            TileObjectData.addTile(Type);
+            Main.tileFrameImportant[Type] = true;
+            Main.tileObsidianKill[Type] = true;
+            Main.tileShine2[Type] = true;
+            Main.tileShine[Type] = 500;
+            Main.tileSpelunker[Type] = true;
+        }
+
+        public override ushort GetMapOption(int i, int j) => (ushort)(Main.tile[i, j].TileFrameX / 18);
+
+        public override void KillTile(int i, int j, ref bool fail, ref bool effectOnly, ref bool noItem)
+        {
+            int toDrop = 0;
+            switch (Main.tile[i, j].TileFrameX / 18)
```

Comment:
may as well just make this an if statement. You probably want to return if `fail` as well

Reply:
```
TileObjectData.newTile.CoordinateWidth = 16; // Width of tile on spritesheet
            TileObjectData.newTile.CoordinateHeights = new int[] { 16 }; // Sets an offset
            TileObjectData.newTile.CoordinatePadding = 2; // Sets an "empty space" that should be between each 16x16 sprite in spritesheet
```
would that be alright?

Reply:
Can't this just be done in the static defaults like example block?
https://github.com/tModLoader/tModLoader/blob/1.4/ExampleMod/Content/Tiles/ExampleBlock.cs#L15

Reply:
yeah, it can be